tables/en-ueb-g2.ctb

```diff
@@ -660,6 +660,8 @@ match %[^_]|%[^_~]%<*[([{] be %[^_]|[)}\\]]%>*%[^_~] 23
 word enough's 26-3-234
 word enough’s 26-3-234
 match %[^_~]([\\([{]*|[\\([{]+%<*) enough (%>*[\\)\\]}]+|[\\)\\]}]*)%[^_~] 26
+word -enough 36-26-1256-126     needed for a test file
+word enough- 26-1256-126-36     needed for a test file
```

Comment:
What does "needed for a test file" mean? Is there a test that exercises this rule? What for?

Reply:
I think it's safe to remove these comments.

Reply:
OK, I'll remove those